### ISSUE TITLE
outerproduct""innerproduct

### ISSUE THREAD

**Issue by ngn** [Dyalog/ride#573]
### Describe the issue you are having
double-backquote completion shows [`outerproduct""innerproduct`](https://i.imgur.com/JVWKEWO.png)

### Did you connect to an already running interpreter or start the interpreter from RIDE?
start

### How do you reproduce the issue?
type ``` ``ou```

### Paste the contents of Help → About (Shift+F1)
```
IDE:
  Version: unknown
  Platform: Linux x86_64
  Date: unknown
  Git commit: unknown
  Preferences:{
    "kbdLocale":"en_US"
  }

Interpreter:
  Version: 15.0.29007
  Platform: Linux-64
  Edition: Unicode/64
  Date: Jan 17 2017 at 01:52:21
```

**Comment by dyaandys** [comment]
The source for this text is svn/trunk/apl/ime/win/src/backtick.entries, and the procedure for updating the RIDE repository from this file is described on the Dyalog internal wiki .. New_Glyphs.

**Comment by e9gille** [comment]
Fixed in d101c7cdc2712ed58882cf25503c4ecbef7f190b
